code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -195,7 +195,8 @@
 		/obj/item/plant_analyzer,
 		/obj/item/cultivator,
 		/obj/item/hatchet,
-		/obj/item/reagent_containers/glass/bottle,
+//		/obj/item/reagent_containers/glass/bottle,
```

Comment:
If you remove stuff then it's best to just plain remove it instead of comment it out. Git(hub) has history itself